citations.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: default
+---
+
+# Galaxy EU citations
```

Comment:
Maybe here a little bit more text? That this is a list of papers that cite the European Galaxy instance ....